In SGXLKL Environment, the munmap() returned 0, but the errno is set to 
correct value i.e. EINVAL. Hence, modified the test check to verify both 
Test return and Test error number. 

diff --git a/testcases/kernel/syscalls/munmap/munmap03.c b/testcases/kernel/syscalls/munmap/munmap03.c
index 3b88b531e..38a556c12 100644
--- a/testcases/kernel/syscalls/munmap/munmap03.c
+++ b/testcases/kernel/syscalls/munmap/munmap03.c
@@ -93,7 +93,7 @@ static void setup(void)
 
 static void check_and_print(int expected_errno)
 {
-	if (TEST_RETURN == -1) {
+	if (TEST_RETURN || TEST_ERRNO) {
 		if (TEST_ERRNO == expected_errno) {
 			tst_resm(TPASS | TTERRNO, "failed as expected");
 		} else {
